subcommand/acl-controller/command_ent_test.go

```diff
@@ -36,3 +36,19 @@ func TestUpsertConsulResourcesEnt(t *testing.T) {
 		},
 	})
 }
+
+func TestUpsertAnonymousTokenPolicyEnt(t *testing.T) {
+	testUpsertAnonymousTokenPolicy(t, map[string]anonTokenTest{
+		"mgw WAN fed enabled, primary DC, create policy": {
+			agentConfig: AgentConfig{
+				Config: Config{Datacenter: "dc1"},
+				DebugConfig: Config{
+					PrimaryDatacenter:               "dc1",
+					MeshGatewayWANFederationEnabled: true,
```

Comment:
Updated! I renamed some of the tests. See what you think!

Reply:
:+1: for the renaming.

Sorry for any confusion, I wasn't very clear about my comment for the `MeshGatewayWANFederationEnabled` field. I was trying to say that we don't actually use this field in the code anymore. And since we don't need it, maybe we should remove it from both the ACL controller and the tests. As you mentioned before it's in the `DebugConfig` part of the agent/self response, so it's not "stable". So it may just be better to get rid of it.

Reply:
Oh, I meant to remove the field, too (forgot to `git add` again before committing)